app/resources/[slug]/page.tsx

```
@@ -73,16 +137,30 @@ export async function generateMetadata(
+    alternates: {
+      canonical: articleUrl,
+    },
   };
 }
 
+// Add script component for JSON-LD
+function JsonLd({ data }: { data: any }) {
+  return (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify(data) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)